Configuration/PyReleaseValidation/python/relval_upgrade.py

```diff
@@ -47,7 +47,8 @@ def makeStepName(key,frag,step,suffix):
                         if 'ProdLike' in specialType:
                             if 'Reco' in step: # handles both Reco and RecoGlobal
                                 stepList[specialType].append(stepMaker(key,frag[:-4],step.replace('RecoGlobal','MiniAOD').replace('Reco','MiniAOD'),specialWF.suffix))
-                                stepList[specialType].append(stepMaker(key,frag[:-4],step.replace('RecoGlobal','Nano').replace('Reco','Nano'),specialWF.suffix))
+                                if 'PU' not in step:
```

Comment:
just for my own understanding, can you explain why Reco+Nano are combined in noPU workflows, but separate in PU workflows?